Dockerfile

```diff
@@ -17,18 +9,27 @@ RUN cargo build --workspace --release \
 
 FROM debian:stretch-slim
```

Comment:
Not sure about this (I know it's after my suggestion): using stretch (debian 9) here, buster (debian 10) in builder.

Reply:
Oh right, now that we depend on `kisiodigital/rust-ci`, we depend on `buster`, not `stretch` anymore. I would say it'd be safer to use a `buster` here. Do you say any risk @datanel @ArnaudOggy?

Reply:
Now using `stretch` everywhere to avoid making more modification that the primary focus of this PR (see 367d943380d1fcae67d847efebbb1259f4dcef83).